source/mdcii/mdcii/include/mdcii/gam/doc/scene.md

```diff
@@ -0,0 +1,115 @@
+The final result after loading the scene file is a savegame file (`.gam`). 
+
+## SZENE_PLAYERMIN
+
+## SZENE_PLAYERMAX
+
+## SZENE_GAMEID
+
+## SZENE_RANKING
+
+### Structs
+
+The main structure is `SceneRanking`.
+
+| SceneRanking | Type  | Description                                 |
+| ------------ | ----- | ------------------------------------------- |
+| ranking      | int32 | The ranking of the missions (stars): 0 to 3 |
+
+## SZENE
+
+The main structure is `SceneSaveData` and contains all other structures and enums listed below.
+
+### Enums
+
+| ClimateType | Type  | Enum   | Value |
+| ----------- | ----- | ------ | ----- |
+|             | uint8 | North  | 0     |
+|             |       | South  | 1     |
+|             |       | Random | 2     |
+
+| SizeType | Type  | Enum  | Value |
+| -------- | ----- | ----- | ----- |
+|          | uint8 | Size0 | 0     |
+|          |       | Size1 | 1     |
+|          |       | Size2 | 2     |
+|          |       | Size3 | 3     |
+|          |       | Size4 | 4     |
+
+| NativeFlag | Type  | Enum      | Value |
+| ---------- | ----- | --------- | ----- |
+|            | uint8 | NoNatives | 0     |
+|            |       | Natives   | 1     |
+
+| GoodsHouseId | Type   | Enum             | Value |
+| ------------ | ------ | ---------------- | ----- |
+|              | uint16 | None             | 0     |
+|              |        | Treasure         | 533   |
+|              |        | OreMine          | 2401  |
+|              |        | GoldMine         | 2405  |
+|              |        | TabacoPlantation | 1336  |
+|              |        | WinePlantation   | 1344  |
+|              |        | SugarPlantation  | 1340  |
+|              |        | CacaoPlantation  | 1338  |
+|              |        | WoolPlantation   | 1332  |
+|              |        | SpicesPlantation | 1342  |
+
+### Structs
+
+| RandomGood | Type         | Description                               |
+| ---------- | ------------ | ----------------------------------------- |
+| houseId    | GoodsHouseId | specifies the houseId that can build this |
+| amount     | uint16       | specifies how many resources there are    |
+| -          | union uint32 | unused? kind, price, percent              |
+
+| RandomNativeVillages | Type      | Description                                |
+| -------------------- | --------- | ------------------------------------------ |
+| strawRoofCount       | uint32    | specifies the count of straw roof villages |
+| incasCount           | uint32    | specifies the count of incas villages      |
+| empty                | uint32[3] | empty                                      |
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/manual/armin.schlegel/mdcii-engine/pullRequest?prid=5498758)